wire-runtime/build.gradle.kts

```diff
@@ -12,6 +13,7 @@ kotlin {
   jvm {
     withJava()
   }
+  jvmToolchain(11)
```

Comment:
1.9.20 is pickier about the JVM compiler and the kotlin compiler being the same, so some of these have been made explicit so that they match.